cmd/object-api-interface.go

```diff
@@ -119,4 +119,7 @@ type ObjectLayer interface {
 	SetBucketLifecycle(context.Context, string, *lifecycle.Lifecycle) error
 	GetBucketLifecycle(context.Context, string) (*lifecycle.Lifecycle, error)
 	DeleteBucketLifecycle(context.Context, string) error
+
+	// Gateway related metrics
+	GetGWMetrics(ctx context.Context) (*GatewayMetrics, error)
```

Comment:
Can you just name this as GetMetrics() which can be extended to XL and FS backend eventually as well?

Reply:
sure..will change

Reply:
done